File: spring-boot-security-jwt-auth-mongodb-master/src/main/java/com/knf/dev/request/SignupRequest.java
package com.knf.dev.request;

import java.util.Set;

public class SignupRequest {

	private String employeename;

	private String email;

	private Set<String> roles;

	private String password;

	public String getEmployeename() {
		return employeename;
	}

	public void setEmployeename(String employeename) {
		this.employeename = employeename;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}

	public Set<String> getRoles() {
		return this.roles;
	}

	public void setRole(Set<String> roles) {
		this.roles = roles;
	}
}

File: kotlin-spring-main/kotlin-springsecurity-jwt/src/main/kotlin/com/knf/dev/demo/kotlinspringsecurityjwt/request/SignupRequest.kt
package com.knf.dev.demo.kotlinspringsecurityjwt.request

import javax.validation.constraints.Email
import javax.validation.constraints.NotBlank
import javax.validation.constraints.Size

class SignupRequest {
    var employeename: @NotBlank @Size(min = 3, max = 20) String? = null
    var email: @NotBlank @Size(max = 50) @Email String? = null
    var roles: Set<String>? = null
        private set
    var password: @NotBlank @Size(min = 6, max = 40) String? = null
    fun setRole(roles: Set<String>?) {
        this.roles = roles
    }
}